testing/scenario_app/android/app/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="dev.flutter.scenarios">
+    <uses-permission android:name="android.permission.INTERNET"/>
+
+    <application
+        android:name="io.flutter.app.FlutterApplication"
+        android:allowBackup="true"
+        android:icon="@mipmap/ic_launcher"
+        android:label="@string/app_name"
+        android:roundIcon="@mipmap/ic_launcher_round"
+        android:supportsRtl="true"
+        android:theme="@style/AppTheme">
+        <activity
+            android:name=".MainActivity"
```

Comment:
I'm not entirely clear yet on exactly where the tests are executed in this project setup, but one thing to make sure is that we can support different Activitys for different tests. For example, if we only support one definition of a main Activity, then this project should be setup in a way that makes it trivial to have that Activity just be a list of buttons to other Activitys that run various use-cases. Or, if that can't be done, then we should find a way to allow for multiple projects to exist here. But I think if we're assuming that a singular MainActivity can exist for all such use-cases and tests, I'm pretty sure that will come up short rather quickly.

Reply:
This is something we can do as we iterate on the project and create actual tests to run. For now, this is just to get something that can be run and used to verify the embedding.